<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Automation Test Suite" duration-ms="15202" started-at="2020-09-11T17:45:58Z" finished-at="2020-09-11T17:46:14Z">
    <groups>
    </groups>
    <test name="Test Cases" duration-ms="15202" started-at="2020-09-11T17:45:58Z" finished-at="2020-09-11T17:46:14Z">
      <class name="com.qa.testcases.HomePageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.testcases.HomePageTest@2133c8f8]" name="setUp" is-config="true" duration-ms="10563" started-at="2020-09-11T23:15:58Z" finished-at="2020-09-11T23:16:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="seatBooking()[pri:0, instance:com.qa.testcases.HomePageTest@2133c8f8]" name="seatBooking" duration-ms="4603" started-at="2020-09-11T23:16:09Z" finished-at="2020-09-11T23:16:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seatBooking -->
        <test-method status="FAIL" signature="tearDown()[pri:0, instance:com.qa.testcases.HomePageTest@2133c8f8]" name="tearDown" is-config="true" duration-ms="0" started-at="2020-09-11T23:16:14Z" finished-at="2020-09-11T23:16:14Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.qa.testcases.HomePageTest.tearDown(HomePageTest.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:709)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.testcases.HomePageTest -->
    </test> <!-- Test Cases -->
  </suite> <!-- Automation Test Suite -->
</testng-results>
